src/api/.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2018-06-20 13:49:50 +0000 using RuboCop version 0.57.2.
+# on 2018-06-20 14:26:57 +0000 using RuboCop version 0.57.2.
```

Comment:
why is this in the [_Add rake task for updating rubocop todo files_ commit](https://github.com/openSUSE/open-build-service/pull/5173/commits/e3673bc54c91a7b3d483cb9d4c00b5fe0b5ed2cf)? :confused: